org.osgi.test.cases.feature/bnd.bnd

```diff
@@ -0,0 +1,26 @@
+# Set javac settings from JDT prefs
+-include: ${build}/eclipse/jdt.bnd, ${build}/osgi.tck.junit-platform.bnd, ${build}/cmpn.bnd
+
+-conditionalpackage					= org.osgi.test.support.*
+-privatepackage						=  \
+	${p}.*
+Import-Package: ${-signaturetest}, *
+
+-buildpath: \
+	org.osgi.test.support;version=project,\
+	org.osgi.service.feature;version=latest,\
+	osgi.core;maven-scope=provided;version='4.3.1',\
+	org.json;version='1.0.0',\
+	org.apache.geronimo.specs.geronimo-json_1.1_spec
+
+-runbundles						=\
+	org.osgi.impl.bundle.serviceloader;version=latest,\
+	org.osgi.impl.service.feature; version=latest,\
+	org.osgi.service.feature; version=latest,\
```

Comment:
You should move this line into the lib file.

Reply:
done